src/sentry/debug_files/artifact_bundles.py

```diff
@@ -188,17 +192,34 @@ def maybe_renew_artifact_bundles_from_processing(project_id: int, used_download_
             continue
         artifact_bundle_ids.append(ty_id)
 
-    # FIXME: This function is being called for every processed event, so ideally
-    # we would heavily debounce this and avoid doing such a query directly.
+    redis_client = get_redis_cluster_for_artifact_bundles()
+
+    redis_client.sadd(get_refresh_key(), *artifact_bundle_ids)
```

Comment:
I believe this is what you mean by "cache key"? Here we are adding all the IDs to the set that symbolicator reported as being used.

Reply:
Perfect, I missed it! Thanks